tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Pricing/CataloguePriceSteps.cs

```diff
@@ -19,9 +18,10 @@ internal sealed class CataloguePriceSteps
         private readonly Response _response;
         private readonly ScenarioContext _context;
 
-        private const string pricingUrl = "http://localhost:5200/api/v1/solutions/{0}/prices";
+        private const string getPriceBySolutionIdUrl = "http://localhost:5200/api/v1/solutions/{0}/prices";
+        private const string getPriceByCatalogueItemIdUrl = "http://localhost:5200/api/v1/prices?catalogueItemId={0}";
```

Comment:
getPricesUrlTemplate? / listPricesUrlTemplate? Might need to modify the constant slighty & introduce a method to get the correct URL dependant on whether or not a catalogueItemId is supplied.